tests/config/run-format-test.js

```diff
@@ -577,18 +577,27 @@ const externalPlugins = new Map([
   ["oxc-ts", "plugin-oxc"],
   ["hermes", "plugin-hermes"],
 ]);
+const loadedPlugins = new Map();
+function loadPlugin(pluginName) {
+  if (loadedPlugins.has(pluginName)) {
+    return loadedPlugins.get(pluginName);
+  }
+  const url = new URL(
+    isProduction
+      ? `../../dist/${pluginName}/index.mjs`
+      : `../../packages/${pluginName}/index.js`,
+    import.meta.url,
+  );
+  const plugin = TEST_STANDALONE ? import(url) : url;
+  loadedPlugins.set(pluginName, plugin);
```

Comment:
technically we should cache the version too, you can have two plugins with different versions, most likely it will never happen but it would be a smarter solution

Reply:
This is wrong, we can't cache by name. Production and dev test are different.